test/functional/test_framework/messages.py

```diff
@@ -31,7 +31,9 @@
 MY_SUBVERSION = b"/python-mininode-tester:0.0.3/"
 MY_RELAY = 1 # from version 70001 onwards, fRelay should be appended to version messages (BIP37)
 
-MAX_INV_SZ = 50000
+# TODO: Add a test that uses MAX_INV_SZ - e.g. tests that this is the maximum number of INVs returned
```

Comment:
I think it would be easier if you just added that test. Would be trivial to send a getdata or inv with that many entries and assert that we don't hear back from them.